reference/7.1/PSReadLine/About/about_PSReadLine.md

```diff
@@ -187,6 +187,19 @@ Deletes the previous requested logical lines and the current logical line in a m
 
 - Vi command mode: `<d,k>`
 
+### DeleteRelativeLines
+
+Deletes from the beginning of the buffer to the current logical line in a multiline buffer.
```

Comment:
Quoted from the PR: https://github.com/PowerShell/PSReadLine/pull/1752
> This PR introduces a new function DeleteRelativeLines **which deletes any number of lines before or after the current logical line in a multiline buffer.**

So maybe it should be _any number of lines of the buffer_ instead of _the beginning of the buffer_.